### PR TITLE
Doc for blueprint

### PR DESCRIPTION
**Pull request by ulziibay**
Screen Shot 2020-01-09 at 12 30 33 PM
Screen Shot 2020-01-09 at 12 30 20 PM

### CODE REVIEW

File: docs/flake8-flask/use-blueprint-for-modularity.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+# r2c-flask-use-blueprint-for-modularity
+
+**tl;dr**: This check recommends using Blueprint when there are too many route handlers in a single file. Using Blueprint encourages modularity and [can greatly simplify how large applications work and provide a central means for Flask extensions to register operations on applications.](https://flask.palletsprojects.com/en/1.1.x/blueprints/#blueprints)
```
Comment (mschwager) [inline docs/flake8-flask/use-blueprint-for-modularity.md]: "Using Blueprint encourages" -> "Blueprint encourages"

File: docs/flake8-flask/use-blueprint-for-modularity.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+## Description
+The check fires when there are more than 3 routes with same prefix in a file.
```
Comment (mschwager) [inline docs/flake8-flask/use-blueprint-for-modularity.md]: "with same prefix" -> "with the same prefix"

File: docs/flake8-flask/use-blueprint-for-modularity.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+definitions that are very long and the user does not seem to be using blueprints!
```
Comment (mschwager) [inline docs/flake8-flask/use-blueprint-for-modularity.md]: "definitions" -> "Definitions"

File: docs/index.md [hunk 4]
```diff
@@ -30,6 +30,11 @@ Flask will not autoescape Jinja templates that do not use `.html`, `.htm`, `.xml
 
 `flask.jsonify()` is a [Flask](https://palletsprojects.com/p/flask/) helper method which handles the correct settings for returning JSON from Flask routes. This check catches uses of `json.dumps()` returned from Flask routes and encourages `flask.jsonify()` instead.
 
+[**r2c-flask-use-blueprint-for-modularity**](flake8-flask/use-blueprint-for-modularity)
+
```
Comment (mschwager) [inline docs/index.md]: Nit: you've got an extra newline here compared to the above links.

File: docs/index.md [hunk 5]
```diff
@@ -30,6 +30,11 @@ Flask will not autoescape Jinja templates that do not use `.html`, `.htm`, `.xml
...
+
+This check recommends using Blueprint when there are too many route handlers in a single file. Using Blueprint encourages modularity and [can greatly simplify how large applications work and provide a central means for Flask extensions to register operations on applications.](https://flask.palletsprojects.com/en/1.1.x/blueprints/#blueprints)
```
Comment (mschwager) [inline docs/index.md]: "Using Blueprint encourages" -> "Blueprint encourages"

File: docs/index.md (hunk 4 above)
Comment (ulziibay) [reply docs/index.md]: Ack

**Pull request merged**